website/docs/developer/research/research-lambda-container.md

```diff
@@ -148,6 +148,152 @@ Based on this research, it looks like using lambda as containers is possible whe
+
+- Use an alternative Dockerfile. The example below packages two code files into a lightweight python container with a `CMD` to run the main file.
+  ```yaml
+  FROM python:3.8-slim-buster
+
+  WORKDIR /src
+
+  # Copy function code
+  COPY main.py main.py
+  COPY sqs_library.py sqs_library.py
+
+  # Install the function's dependencies using file requirements.txt
+  # from your project folder.
+
+  COPY requirements.txt  .
+  RUN  pip3 install -r requirements.txt
+  COPY . .
+
+  CMD ["python", "main.py"]
+  ```
+- Follow prior instructions up to the Terraform deployment. Use the following Terraform instead of the pervious example.
+```terraform
+data "aws_iam_policy_document" "sqs_task_policy_document" {
+  statement {
+    actions   = ["sts:AssumeRole"]
+    resources = ["*"]
+  }
+  statement {
+    actions = [
+      "sqs:ReceiveMessage",
+      "sqs:SendMessage",
+      "sqs:DeleteMessage",
+      "sqs:GetQueueAttributes"
+    ]
+    resources = ["*"]
+  }
+}
+
+data "aws_iam_policy_document" "assume_ecs_tasks_role_policy_document" {
+  statement {
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+# IAM role that tasks can use to make API requests to authorized AWS services.
+resource "aws_iam_role" "orca_ecs_tasks_role" {
+  name                 = "${var.prefix}_orca_ecs_tasks_role"
+  assume_role_policy   = data.aws_iam_policy_document.assume_ecs_tasks_role_policy_document.json
+  permissions_boundary = var.permissions_boundary_arn
+  tags                 = var.tags
+}
+
+resource "aws_iam_role_policy" "sqs_task_role_policy" {
+  name   = "${var.prefix}_orca_sqs_task_role_policy"
+  role   = aws_iam_role.orca_ecs_tasks_role.id
+  policy = data.aws_iam_policy_document.sqs_task_policy_document.json
+}
+
+# This role is required by tasks to pull container images and publish container logs to Amazon CloudWatch on your behalf.
+resource "aws_iam_role" "orca_ecs_task_execution_role" {
+  name                 = "${var.prefix}_orca_ecs_task_execution_role"
+  assume_role_policy   = data.aws_iam_policy_document.assume_ecs_tasks_role_policy_document.json
+  permissions_boundary = var.permissions_boundary_arn
+  tags                 = var.tags
+}
+
+
+resource "aws_iam_role_policy_attachment" "ecs_role_policy" {
+  role       = aws_iam_role.orca_ecs_task_execution_role.name
+  policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
+}
+
+resource "aws_ecs_cluster" "test-cluster" {
+  name = "${var.prefix}_orca_ecs_cluster"
+  capacity_providers = ["FARGATE"]
+  default_capacity_provider_strategy {
+    capacity_provider = "FARGATE"
+    weight            = 100
+  }
+}
+
+resource "aws_ecs_task_definition" "task" {
+  family                   = "${var.prefix}_orca_sqs_task"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "4096"
```

Comment:
using larger cpu will add in cost. We need to decide a proper value in case fargate is used

Reply:
Added note.